### PR TITLE
MES-6426 deleted un needed files for first test scenario

### PR DESCRIPTION
**Pull request by turnerc1**
## Checklist

- [x] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: test/e2e/tsconfig.e2espec.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      "es2016"
+    ],
+    "outDir": "../dist/out-tsc-e2e",
+    "module": "commonjs",
+    "target": "es6",
+    "types":[
+      "node"
+    ]
+  }
+}
```
Comment (Cullima) [inline test/e2e/tsconfig.e2espec.json]: missing empty line

**Review by Cullima — APPROVED** [review]

File: test/e2e/tsconfig.e2espec.json (hunk 1 above)
Comment (turnerc1) [reply test/e2e/tsconfig.e2espec.json]: Added blank line

File: package.json [hunk 2]
```diff
@@ -22,7 +22,9 @@
     "stylelint": "stylelint 'src/**/*.scss' fix --config stylelint.json",
     "gitSecrets": "git secrets --scan",
     "scanRepo": "git log -p -n 15 | scanrepo",
-    "e2e": "ng e2e",
+    "test:e2e": "protractor e2e-simulator-bdd.conf.js",
+    "test:generate-report": "node generateTestReport.js",
+    "e2e-build": "xcodebuild -workspace ios/App/App.xcworkspace -scheme App -sdk iphonesimulator SYMROOT=$(PWD)/build",
```
Comment (andrewsetterfield) [inline package.json:27]: we probably want to add an additional command in here to make sure it uses the correct config file.

npm run config:dev && xcodebuild -workspace ios/App/App.xcworkspace -scheme App -sdk iphonesimulator SYMROOT=$(PWD)/build"

File: test/e2e/step-definitions/generic-steps.ts [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+  });
+});
+
+/// ///////////////////////////////////////// SHARED FUNCTIONS ////////////////////////////////////////////
+
+/**
+ * Output the page source to a file - For debug purposes only
+ * @param fileName the name of the file to output to
+ */
+export const logPageSource = (fileName: string) => {
```
Comment (andrewsetterfield) [inline test/e2e/step-definitions/generic-steps.ts]: are you sure this is used?

File: test/e2e/tsconfig.e2espec.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "sourceMap": true,
+    "declaration": false,
+    "moduleResolution": "node",
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "lib": [
+      "es2016"
```
Comment (andrewsetterfield) [inline test/e2e/tsconfig.e2espec.json]: can we try es2018 to match des4.

File: test/e2e/step-definitions/generic-steps.ts (hunk 3 above)
Comment (turnerc1) [reply test/e2e/step-definitions/generic-steps.ts]: Removed

File: test/e2e/tsconfig.e2espec.json (hunk 4 above)
Comment (turnerc1) [reply test/e2e/tsconfig.e2espec.json]: Changed to 2018, nothing changed from what I can see

**Review by andrewsetterfield — APPROVED** [review]

**Pull request merged**